agent/tests/conftest.py

```diff
@@ -90,6 +90,68 @@ def one_configuration_row_lmx():
     )
 
 
+@fixture
```

Comment:
Docstrings are missing in some fixtures here

Reply:
I added docstrings to all fixtures in the latest commit